src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -1449,6 +1455,12 @@ namespace Opm {
         }
     }
 
+    void Schedule::InvalidWellPattern( const std::string& wellNamePattern,  const ParseContext& parseContext, const DeckKeyword& keyword ) const {
```

Comment:
This functionality is not related to the content of the  Schedule class - I recommend just making a loose function in an anonymous namespace.

Reply:
used in `Schedule/Completion.cpp`

Reply:
Ok